cli/command/container/opts.go

```diff
@@ -481,12 +484,16 @@ func parse(flags *pflag.FlagSet, copts *containerOptions) (*containerConfig, err
 		if copts.healthRetries < 0 {
 			return nil, fmt.Errorf("--health-retries cannot be negative")
 		}
+		if copts.healthStartPeriod < 0 {
+			return nil, fmt.Errorf("--health-start-period cannot be negative")
```

Comment:
since docker daemon would expose this in the api, thus I think we need to update this parameter in the daemon side as well, WDYT? @elifa